valpas-web/src/state/oppijat.ts

```diff
@@ -90,3 +120,19 @@ export const Haku = {
   latest: (haut: Haku[]) =>
     pipe(haut, A.sortBy([muokkausOrd]), A.head, O.toNullable),
 }
+
+export const Yhteystiedot = {
+  isIlmoitettu: (
```

Comment:
Aika varman käsitykseni mukaan käyttöliittymässä "ilmoitetutut yhteystiedot" ovat niitä, mitkä opo on ilmoittanut kunnalle virallisen oppivelvollisuusilmoituksen yhteydessä.

Nyt oppijan itse hakemukselleen kirjaamat yhteystiedot menevät niiden kanssa sekaisin sekä käyttöliittymässä että tässä toteutuksessa.

Kälikuvissa hakeutumisen valvojalle ei ole piirretty lainkaan oppijan itse hakemukselleen ilmoittamien tietojen näyttämistä perusnäkymän oikeassa ylälaidassa. Oppijan itse hakemukselleen ilmoittamia tietoja on kälispeksien mukaan tarkoitus näyttää ja käyttää ainoastaan pohjatietoina opon tekemässä ilmoituksessa, mitä me ei ole vielä toteutettu. Eli selkeintä olisi varmaan tässä vaiheessa poistaa koko tämä hakemukselta tulevien yhteystietojen näyttäminen ja näyttää pelkät VTJ-tiedot oppijanäkymässä.

Reply:
Pilotin kannalta: jos nämä hakulomakkeen yhteystiedot poistetaan jo bäkkärillä, eikä välitetä frontille asti lainkaan, niin sitten meidän ei tarvitse sitä turvakielto-notifikaatiotakaan pilottia varten toteuttaa.

Reply:
Hmm, eikun onhan ne tuonne kälikuviin piirretty "Ilmoiteutut yhteystiedot" alle.

![image](https://user-images.githubusercontent.com/6376268/112271835-1755c200-8c84-11eb-942b-ad5a13785086.png)

Mielestäni ne on aika epäselvä kyllä näyttää tuon otsikon alla, jos ei ole tuota kälikuviin piirrettyä selvennystä yhteystietojen lähteestä mukana.

Reply:
Tää on nyt vähän epäselvä. Ehkä tosiaan paras, että ei palauteta yhteystietoja ollenkaan tässä vaiheessa, ja Tommin palatessa lomilta tämä mietitään tarkemmin.

Tämän kuvan voi kuitenkin tulkita niinkin, että haun yhteydessä annetut yhteystiedot näytettäisiin.

![image](https://user-images.githubusercontent.com/6116633/112437586-c9a68b80-8d4f-11eb-8695-2a94bde9e151.png)

Reply:
Nyt tässä pullarissa oleva toteutus on näin:

* Huoltajan tietoja ei näytetä (eikä palauteta APIsta)
* Ilmoitettuihin yhteystietoihin on lisätty lähde ja päivämäärä
* Virallisia yhteystietoja ei haeta, jos oppijalla on turvakielto